### PR TITLE
Remove fields that are not used or unreachable from the entry points.

### PR DESCRIPTION
**Pull request by arturobernalg**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [X] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Description

<!--

### CODE REVIEW

File: liquibase-core/src/main/java/liquibase/command/core/DropAllCommand.java [hunk 1]
```diff
@@ -105,8 +105,6 @@ public CommandResult run() throws Exception {
         commandScope.addArgumentValue(InternalDropAllCommandStep.HUB_CONNECTION_ID_ARG, this.hubConnectionId);
         commandScope.addArgumentValue(InternalDropAllCommandStep.SCHEMAS_ARG, this.schemas);
 
-        final CommandResults results = commandScope.execute();
```
Comment (nvoxland) [inline liquibase-core/src/main/java/liquibase/command/core/DropAllCommand.java:108]: This line can't be removed, just the output variable

**Review by nvoxland — DISMISSED** [review]
**Initial/Pre-Review Thoughts**

They all look like fields in a function, so we don't have to worry about API changes. 

But, we have to watch out for side-effects from functions if the whole function call is removed vs. just removing the output variable.

For example, the commandScope.execute() call can't be removed. I didn't look closely into any of the other lines, but we should check those.

Questions I have:
- None

Potential risks:
- None

What could make the full review difficult:
- Nothing

File: liquibase-core/src/main/java/liquibase/command/core/DropAllCommand.java (hunk 1 above)
Comment (arturobernalg) [reply liquibase-core/src/main/java/liquibase/command/core/DropAllCommand.java:108]: done. reverted.

File: liquibase-core/src/main/java/liquibase/command/core/DropAllCommand.java [hunk 2]
```diff
@@ -104,8 +104,7 @@ public CommandResult run() throws Exception {
         commandScope.addArgumentValue(InternalDropAllCommandStep.DATABASE_ARG, this.database);
```
Comment (MalloD12) [inline liquibase-core/src/main/java/liquibase/command/core/DropAllCommand.java:104]: @filipelautert this class has been marked as deprecated, but this `run()` hasn't been. Should not we mark it as deprecated as well?

File: liquibase-core/src/main/java/liquibase/sqlgenerator/core/SelectFromDatabaseChangeLogLockGenerator.java [hunk 3]
```diff
@@ -27,7 +27,6 @@ public Sql[] generateSql(SelectFromDatabaseChangeLogLockStatement statement, fin
    		liquibaseSchema = database.getLiquibaseSchemaName();
 		
 		ColumnConfig[] columns = statement.getColumnsToSelect();
```
Comment (MalloD12) [inline liquibase-core/src/main/java/liquibase/sqlgenerator/core/SelectFromDatabaseChangeLogLockGenerator.java:29]: I don't see any use of columns variable either. Removing it too.

**Review by MalloD12 — COMMENTED** [review]
## Review and testing results:

Code changes looks good to me. Don't see or had any concern on the removed fields/unreachable code.

### Things to be aware of:
 - Pro build is failing and that's not allowing to run functional and test harness tests at the moment.

### Things to worry about:
- None

I'll wait to have at least a functional/test harness run before approving this PR.

**Review by MalloD12 — APPROVED** [review]
**Approved**.

Test harness and most of the functional tests jobs are now passing, failing one (mssql) seems to be failing for non-related reason.

**Review by filipelautert — APPROVED** [review]

**Comment by XDelphiGrl** [comment]
Functional tests re-running here: [https://github.com/liquibase/liquibase-pro-tests/actions/runs/4928280271](https://github.com/liquibase/liquibase-pro-tests/actions/runs/4928280271)

File: liquibase-standard/src/main/java/liquibase/command/core/DropAllCommand.java
[the remark this answers is not in the document]
Comment (MalloD12) [reply liquibase-standard/src/main/java/liquibase/command/core/DropAllCommand.java:1]: If I didn't get this wrong, this class has been deleted as part of command refactoring work. 

cc: @filipelautert

File: liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseDeactivateChangeLogMojo.java [hunk 4]
```diff
@@ -20,7 +19,6 @@ public class LiquibaseDeactivateChangeLogMojo extends AbstractLiquibaseChangeLog
     protected void performLiquibaseTask(Liquibase liquibase)
         throws LiquibaseException {
         super.performLiquibaseTask(liquibase);
-        Database database = liquibase.getDatabase();
```
[the remark this answers is not in the document]
Comment (MalloD12) [reply liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseDeactivateChangeLogMojo.java:23]: It has been removed yet in [master](https://github.com/liquibase/liquibase/blob/master/liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseDeactivateChangeLogMojo.java).

[the remark this answers is not in the document]
Comment (XDelphiGrl) [reply liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseDeactivateChangeLogMojo.java:23]: It has now, lol. Thanks for getting back to me, @MalloD12!

[the remark this answers is not in the document]
Comment (MalloD12) [reply liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseDeactivateChangeLogMojo.java]: Done! :D

File: liquibase-standard/src/main/java/liquibase/command/core/DropAllCommand.java
[the remark this answers is not in the document]
Comment (XDelphiGrl) [reply liquibase-standard/src/main/java/liquibase/command/core/DropAllCommand.java:1]: I believe this empty file will go away when master is merged into this PR; we've recently merged the Hub decommission branch to liquibase/liquibase master.

**Review by XDelphiGrl — APPROVED** [review]
This is a code cleanup PR, removing declarations for variables that are never used.
* Functional and test harness executions passing.
* No additional testing required.

APPROVED

File: liquibase-standard/src/main/java/liquibase/command/core/DropAllCommand.java
[the remark this answers is not in the document]
Comment (MalloD12) [reply liquibase-standard/src/main/java/liquibase/command/core/DropAllCommand.java:1]: I just removed this empty file as well (as I did with LiquibaseDeactivatedChangeLogMojo file).

**Pull request merged**